pip-requirements.txt

```diff
@@ -9,7 +9,6 @@ sphinx-argparse
 sphinx_rtd_theme
 sphinx-gallery
 tensorflow>=2.3
-tensorflow-addons
```

Comment:
Can you note in the PR description that there are requirement changes in it (and maybe note what PR removing this package this is in response to?)

Reply:
Added, I forgot to do this step in my last PR.